e2e/tests/LoggedIn/browse.e2e.spec.ts

```diff
@@ -1,28 +1,37 @@
-import { expect, test } from '@playwright/test';
-import * as Helpers from '../helpers';
-import { PageModel } from '../PageModel';
+import { expect, test } from "@playwright/test";
+import * as Helpers from "../helpers";
+import { PageModel } from "../PageModel";
```

Comment:
(Minor) is there a reason for the change? /api & /ui have single quotes (via prettier)
It would be nice to be consistent across project where possible.

Reply:
it seems my prettier puts double quotes for imports and single quotes for strings

Reply:
I'm using the default settings for prettier

Reply:
Yeah, it seems the person who worked before had some custom prettier settings, that's why.

Reply:
no probs - there's no prettier rule set up atm in /e2e, just /api /ui